AccessibilityCodelab/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt

```diff
@@ -165,11 +146,11 @@ fun PostCardPopular(
     modifier: Modifier = Modifier
 ) {
     Card(
-        shape = MaterialTheme.shapes.medium,
-        modifier = modifier.size(280.dp, 240.dp),
         onClick = { navigateToArticle(post.id) },
+        shape = MaterialTheme.shapes.medium,
         // Step 2: Click labels
-        onClickLabel = stringResource(id = R.string.action_read_article)
+//        onClickLabel = stringResource(id = R.string.action_read_article),
```

Comment:
Could you help me figure out the correct change here? cc @JolandaVerhoef

Reply:
What's the failure?

Reply:
Won't compile with DeprecationLevel.ERROR. Suggests alternative constructors that don't have onClickLabel

Reply:
Got it. The codelab will have to be adapted (code and text). I can do that!